src/app/new-page/component/content-spacer/content-spacer.component.css

```diff
@@ -0,0 +1,5 @@
+.spacer {
+  display: block;
+  width: 100%;
+  min-height: 10px;
```

Comment:
let's not do a `min-height`, we will depend on the content admins to define exactly the size they want